internal/core/src/segcore/load_index_c.cpp

```diff
@@ -349,13 +349,14 @@ AppendIndexV2(CTraceContext c_trace, CLoadIndexInfo c_load_index_info) {
         LOG_INFO(
             "[collection={}][segment={}][field={}][enable_mmap={}][load_"
             "priority={}] load index "
-            "{}",
```

Comment:
it seems to me the `mmap_dir_path` argument of `AppendFieldInfo` should also be removed

Reply:
this log should be removed as we will log at line 438

Reply:
AppendFieldInfo has been removed


second print out is for running status clarity when debugging